gnovm/pkg/gnolang/nodes.go

```diff
@@ -502,6 +561,10 @@ type FuncLitExpr struct {
 	Body              // function body
 }
 
+func (x *FuncLitExpr) isAddressable() bool {
+	return false
```

Comment:
this will be updated based on the result type, right?

Reply:
No. This is a function literal, not an execution. Hopefully the tests added here make this more clear https://github.com/gnolang/gno/pull/2731/commits/af12ca140043803d2c4f028d31968cabf67814d2